packages/lib/server/defaultResponder.appDir.ts

```diff
@@ -0,0 +1,34 @@
+import type { NextRequest } from "next/server";
+import { NextResponse } from "next/server";
+
+import { getServerErrorFromUnknown } from "./getServerErrorFromUnknown";
+import { performance } from "./perfObserver";
+
+type Handle<T> = (req: NextRequest) => Promise<T>;
+
+/** Variant to use in App Router API routes. Allows us to get type inference from API handler responses */
+export const defaultResponder =
+  <T>(f: Handle<T>) =>
+  async (req: NextRequest) => {
+    let ok = false;
+    try {
+      performance.mark("Start");
```

Comment:
We can use the Sentry monitoring `monitorCallbackAsync` instead

Reply:
done